src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -44,6 +44,9 @@ const propTypes = {
     /** Does this fragment belong to a reportAction that has not yet loaded? */
     loading: PropTypes.bool,
 
+    /** The reportAction's source */
+    source: PropTypes.oneOf(['Chronos', 'email', 'ios', 'android', 'web', 'email', '']),
```

Comment:
Empty string is a valid source?

Reply:
And should this be a required prop?

Reply:
I think there are places where we don't pass a source, like here:

https://github.com/Expensify/Web-Expensify/blob/2f46bbb7f189948b8aac833fed097a9f690446ca/api.php#L612-L613

https://github.com/Expensify/Web-Expensify/blob/2f46bbb7f189948b8aac833fed097a9f690446ca/lib/ChatBot/Channel/NativeChat.php#L68

Reply:
> And should this be a required prop?

maybe, I'll look into it. I'll also change the server side PR to always include the source if there is one.

Reply:
I see we have this other usage of `ReportActionItemSingle` here: https://github.com/Expensify/App/blob/de3301c92428e7963bb1bf11ca31d8c49a3511de/src/pages/home/report/ReportActionItemSingle.js#L90-L97

and I'm not passing the `source` there. I'm not familiarized with what are we rendering here..

Reply:
It is rendering the name in the conversation:

<img width="719" alt="image" src="https://user-images.githubusercontent.com/87341702/176321346-9b516317-75d1-4110-b17b-31296273b98f.png">


I personally would keep it as **optional** with default `''` instead of doing it **required** and passing `''` here: https://github.com/Expensify/App/blob/de3301c92428e7963bb1bf11ca31d8c49a3511de/src/pages/home/report/ReportActionItemSingle.js#L90-L97

Reply:
Okay, this PR LGTM then 👍